src/Result/AbstractVersionedSoftware.php

```diff
@@ -20,9 +20,12 @@ public function toString(): string
         return implode(' ', array_filter(array($this->family, $this->toVersion())));
     }
 
-    /** @return string */
-    protected function formatVersion(): string
+    /**
+     * @param string ...$args
```

Comment:
Either `string|null` or remove the docblocks entirely (since they are duplicating the type info)

Reply:
Well should we have to remove all docblocks if when all hints are always declared ?

Reply:
I always remove them to avoid duplicate info in my other projects

Reply:
Makes sense!